zebrad/tests/acceptance.rs

```diff
@@ -745,22 +745,8 @@ fn sync_large_checkpoints_mainnet() -> Result<()> {
     Ok(())
 }
 
-/// Test if `zebrad` can sync some larger checkpoints on testnet.
-///
-/// This test does not run by default, see `sync_large_checkpoints_mainnet`
-/// for details.
-#[test]
-#[ignore]
-fn sync_large_checkpoints_testnet() -> Result<()> {
-    sync_until(
-        LARGE_CHECKPOINT_TEST_HEIGHT,
-        Testnet,
-        STOP_AT_HEIGHT_REGEX,
-        LARGE_CHECKPOINT_TIMEOUT,
-        None,
-    )
-    .map(|_tempdir| ())
-}
+// Todo: We had a `sync_large_checkpoints_mainnet` here but it was removed because
```

Comment:
```suggestion
// Todo: We had a `sync_large_checkpoints_testnet` here but it was removed because
```

Reply:
oops, sorry about that and thanks for the fix.